pennylane_lightning/lightning_kokkos/_measurements.py

```diff
@@ -115,3 +115,26 @@ def _process_single_shot(samples):
         return (
             tuple(zip(*processed_samples)) if shots.has_partitioned_shots else processed_samples[0]
         )
+
+    def probs(self, measurementprocess: MeasurementProcess):
+        """Probabilities of the supplied observable or wires contained in the MeasurementProcess.
+
+        Args:
+            measurementprocess (StateMeasurement): measurement to apply to the state
+
+        Returns:
+            Probabilities of the supplied observable or wires
+        """
+        diagonalizing_gates = measurementprocess.diagonalizing_gates()
```

Comment:
Seems we're missing a test here.

Reply:
I do not understand why CodeCov never catch the data from LKokkos :confused: